runtime/tr.source/trj9/z/codegen/J9S390CHelperLinkage.cpp

```diff
@@ -126,6 +126,7 @@ TR::S390CHelperLinkage::S390CHelperLinkage(TR::CodeGenerator * codeGen,TR_S390Li
       setIntegerArgumentRegister(2, TR::RealRegister::GPR4);
       setIntegerArgumentRegister(3, TR::RealRegister::GPR5);
       setIntegerArgumentRegister(4, TR::RealRegister::GPR6);
+      setPreservedRegisterMapForGC(0x0000BFC0);
```

Comment:
We don't have to do this now, but we would get some pretty subtle bugs if the preserved register list was ever modified without updating this constant. We should consider in the future generating this map using the list of preserved registers directly from the linkage rather than encoding it by hand.

Reply:
Makes sense. I have added a comment at this point. Basically what we should have is one function that goes over list of preserved register and prepares register map for GC. I have added this in `TODO` right now.